### ISSUE TITLE
cannot view report of failed scan job that worked on some nodes

### ISSUE THREAD

**Issue by vjeffrey** [chef/automate#3482]
## User Story

right now we have some logic that says if a scan job overall status is failed, show the error button so users can look at the errors.
heres the thing: a scan job includes many nodes. so sometimes we may not be able to scan one node out of the 30 desired, because of incorrect creds or some other reason. in that case, we show the error button, but give the user no way to view the other node reports that were generated by the scan.
it would be nice to be able to view the reports that were executed for the scan job, even if some of the nodes were not scanned.

**Comment by jonong1972** [comment]
What happens if you search the reports with the Scan Job ID, will it still show the reports of the nodes that did pass?  just omitting the failed node?  If that is the case, simple solution would to add the View Report button with a side note or something telling the user some nodes did not scan for whatever reasons.

**Comment by vjeffrey** [comment]
yes, if you search reports with a job id that had both failed nodes and completed reports, it works: https://a2-local-inplace-upgrade-dev.cd.chef.co/compliance/reports/overview?job_id=5fe7bdc0-7ea1-4cf3-9647-1bbba479f47f&end_time=2020-04-23
